pbrd/pbr_vty.c

```diff
@@ -100,6 +100,12 @@ DEFPY(pbr_set_table_range,
 	int ret = CMD_WARNING;
 	const int minrange = 1000;
 
+	if (no) {
+		pbr_nht_set_tableid_range(PBR_NHT_DEFAULT_LOW_TABLEID,
+					  PBR_NHT_DEFAULT_HIGH_TABLEID);
+		return CMD_SUCCESS;
```

Comment:
ranges in the definition should be optional if we ignore them, split the command out into a no form

Reply:
makes sense, will do